packages/@react-spectrum/textfield/test/TextArea.test.js

```diff
@@ -42,21 +41,9 @@ describe('TextArea', () => {
   // user inputs more than two lines of text. To workaround this on user text input the height style of the textarea is set to
   // "auto" to get the scroller to appear, then the height is set equal to scrollHeight to remove the scroller
   // and properly adjust the height of the textarea to match the currently input text
-  it.each`
-    Name                | Component        | props
-    ${'v3 TextArea'}    | ${TextArea}      | ${{isQuiet: true, onChange}}
-    ${'v2 TextArea'}    | ${V2TextArea}    | ${{quiet: true, onChange}}
-  `('$Name quiet variant automatically adjusts its vertical height on change', ({Component, props}) => {
-    let tree = renderComponent(Component, props);
-    let input = tree.getByTestId(testId);
-
-    expect(input.style.height).toBe('');
-    fireEvent.change(input, {target: {value: '15', style: {}}});
-    expect(input.style.height).toBe(`${mockScrollHeight}px`);
-  });
-
```

Comment:
IMO I feel like this test would be good to have still since it tests the change case. It probably could be accomplished by changing the `beforeAll` above to a `beforeEach` and then doing some thing like 
```
    let newScrollHeight = 1000;
    expect(input.style.height).toBe(`${mockScrollHeight}px`);
    Object.defineProperty(HTMLElement.prototype, 'scrollHeight', {configurable: true, value: newScrollHeight});
    fireEvent.change(input, {target: {value: '15', style: {}}});
    expect(input.style.height).toBe(`${newScrollHeight}px`);
```
What do you think?

Reply:
that looks like it'd work, i think that'd be ok. i was getting close to basically testing the browser in the path i was trying to take, this is better, i'll give it a go